tests/devapps/XForms/XForms.iOS/XForms.iOS.csproj

```diff
@@ -51,10 +51,13 @@
     <MtouchArch>ARMv7, ARM64</MtouchArch>
     <CodesignKey>iPhone Developer</CodesignKey>
     <MtouchDebug>True</MtouchDebug>
-    <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
```

Comment:
This will disable keychain access. Is it on purpose?

Reply:
Debug code, removed